Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    commander (4.4.7)
+      highline (~> 2.0.0)
+    database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubis (2.7.0)
-    event_sourcery (0.13.0)
-    event_sourcery-postgres (0.3.0)
-      event_sourcery (>= 0.10.0)
+    erubi (1.7.1)
+    event_sourcery (0.22.0)
+    event_sourcery-postgres (0.8.0)
```

Comment:
### [event_sourcery-postgres](https://github.com/envato/event_sourcery-postgres)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.3.0 → 0.8.0

[_[change-log](https://github.com/envato/event_sourcery-postgres/blob/master/CHANGELOG.md), [source-code](https://github.com/envato/event_sourcery-postgres)_]





<details>
<summary>Commits</summary>

A change of **144** commits. See the full changes on [the compare page](https://github.com/envato/event_sourcery-postgres/compare/v0.3.0...v0.8.0).

These are the first 10 commits:
- (60683c6) [yard doc for schema.rb file](https://github.com/envato/event_sourcery-postgres/commit/60683c6d12836e62c43732de7397443467aa27e4)
- (75cbbf4) [doc for the version](https://github.com/envato/event_sourcery-postgres/commit/75cbbf48e554c313bceaa73f70b5623b4cfa119d)
- (f91e813) [single quoteys where double quoteys not needed](https://github.com/envato/event_sourcery-postgres/commit/f91e813699ebd244320bead998809865cdb5f330)
- (b5805b7) [inline a block](https://github.com/envato/event_sourcery-postgres/commit/b5805b7851b150be11f5663eab763074d2073112)
- (b01103f) [parentheses are preferred for %w](https://github.com/envato/event_sourcery-postgres/commit/b01103f4e7cc24c5affb97b478056358e89ad880)
- (74447b7) [underscore the unused block param](https://github.com/envato/event_sourcery-postgres/commit/74447b7ae5a3393abfee5da3d61a51b412116d35)
- (f517f33) [inline some blocks, change some sqwiggely blocks to dos and …](https://github.com/envato/event_sourcery-postgres/commit/f517f3356c4173c14e63ee293be5788790377c8c)
- (dc48680) [dos and ends instead of sqwiggely bracers](https://github.com/envato/event_sourcery-postgres/commit/dc48680908bf650ec02ef6c7ed9349820e1a369e)
- (99d2d3a) [more dos and ends and whitespace and some alignment](https://github.com/envato/event_sourcery-postgres/commit/99d2d3a480d2d927dc4bbc0b7442e66b183f65d7)
- (118e5cf) [inline if](https://github.com/envato/event_sourcery-postgres/commit/118e5cf376373b2db9b33db9f54acb5a5bfdcd35)


</details>